src/coreclr/pal/inc/unixasmmacrosriscv64.inc

```diff
@@ -43,12 +43,10 @@ C_FUNC(\Name\()_End):
 
 .macro PROLOG_STACK_ALLOC Size
     addi sp, sp, -\Size
-    .cfi_def_cfa sp,\Size
 .endm
 
 .macro EPILOG_STACK_FREE Size
     addi sp, sp, \Size
-    .cfi_def_cfa sp,-\Size
```

Comment:
@clamp03 the `EPILOG_STACK_FREE` should have the `.cfi_def_cfa sp,-\Size`, only the `PROLOG_STACK_ALLOC` should not have it.

Reply:
Thank you. Restored!